src/playlist.js

```diff
@@ -105,11 +105,12 @@
       if (!playlist.endList) {
         return window.Infinity;
       }
+      endSequence = playlist.segments.length;
```

Comment:
I only see it being assigned to here and the comment at the top of this function indicates `playlist.mediaSequence` is already a component. Seems like a bug waiting to happen.